optaplanner-quickstarts/quarkus-school-timetabling/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id "io.quarkus" version "1.4.2.Final"
```

Comment:
This is also a maintenance burden - we will want to update this version on regular basis.

Reply:
Yea, same with the spring version.
But that's part of the game. Not much we can do about it.